sdk/core/Azure.Core/samples/Mocking.md

```diff
@@ -1,55 +1,3 @@
 # Azure.Core mocking samples
 
-**NOTE:** Samples in this file apply only to packages that follow [Azure SDK Design Guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html). Names of such packages usually start with `Azure`. 
-
-## Creating a mock of the client using Moq
-
-You can use a combination of model factory and `Mock` class to create a mock of a client:
-
-```C# Snippet:ClientMock
```

Comment:
I think you will need to remove the corresponding code, otherwise the Update-Snippets command will fail locally.

Reply:
The snippet is still being referenced in the Azure.Core README. I did, however, delete the other snippet in that file.